### ISSUE TITLE
Tour Button Text Contrast SiteNow v2

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#2480]
# Problem

Button contrast issue on tour buttons.

<img width="270" alt="Screen Shot 2020-11-05 at 11 52 44 AM" src="https://user-images.githubusercontent.com/4663676/98277891-7cbb4780-1f5d-11eb-9b0b-f31e26ee8b13.png">

<img width="525" alt="Screen Shot 2020-11-16 at 9 26 53 AM" src="https://user-images.githubusercontent.com/4663676/99273138-3df38000-27ee-11eb-92f7-6a149736cf07.png">

# Proposed solution

- Provide tour specific style override to change the button text to black.

**Comment by joewhitsitt** [comment]
Seeing this on a v2 "standard" site

**Comment by joewhitsitt** [comment]
Looks like v3 handles this with an outline button.
<img width="165" alt="Screen Shot 2020-11-16 at 9 28 47 AM" src="https://user-images.githubusercontent.com/4663676/99273088-2d430a00-27ee-11eb-9ced-f7126e7675af.png">
